tendrl/commons/objects/definition/master.yaml

```diff
@@ -111,6 +127,36 @@ namespace.tendrl:
       list: /clusters
       value: clusters/$TendrlContext.integration_id
       atoms:
+        StopNodeServices:
```

Comment:
break this down into  StopMonitoringServices and StopIntegrationServices

Reply:
Done